workers/datasets_based/src/datasets_based/worker_loop.py

```diff
@@ -97,6 +98,8 @@ def sleep(self) -> None:
         time.sleep(duration)
 
     def loop(self) -> None:
+        if os.environ.get("HF_ENDPOINT"):
```

Comment:
We don't access the os environment variables outside of the config.py files. Do you think it's possible to keep on with this practice and move the log there?

Reply:
I kept the logging here and got the endpoint value from worker_factory.app_config.common

Reply:
Perfect, thanks!